src/reason_oprint.ml

```diff
@@ -47,6 +47,7 @@
  *
  *)
 
+open Ast_404
```

Comment:
The conversion is done when plugging to the Toplevel: in `Reason_utop`, printing functions are lifted.